plugins/wme-sitebuilder/wme-sitebuilder.php

```diff
@@ -50,6 +50,7 @@
 	]);
 
 	$wme_sitebuilder->init();
+	$wme_sitebuilder->set_admin_color_scheme();
```

Comment:
Any reason this can't just be in the `init()` method after `add_admin_color_scheme()` if it's going to run on every request?

Reply:
That makes perfect sense @defunctl, moving it.